lib/esm/index.js

```diff
@@ -0,0 +1,32 @@
+import { BorshSerializer } from './serialize';
+import { BorshDeserializer } from './deserialize';
+import * as utils from './utils';
+import bs58 from 'bs58';
```

Comment:
I just realized this, this will be problematic as bs58 is not packaged in esm. So even if this is packaged in esm near-sdk-js will not able to import or use it. Could we drop base_encode and base_decode for esm packaging until we find an esm packaged bs58 impl?

Reply:
removed bs58